add new minimap

Royalot opened the pull request:
![minimap_floor](https://cloud.githubusercontent.com/assets/6708725/6971325/151e0e8c-d97c-11e4-8887-4f8015cc6cee.png)

Royalot committed:
add new minimap

![minimap_floor](https://cloud.githubusercontent.com/assets/6708725/6902333/71bf6318-d713-11e4-877a-aff5889b62d5.png)

Schiffers commented:
update this, now have the surrenders dreams :3

tarantonio commented:
and calcanea is there too

dbjorkholm commented:
Treasure Hunt Island, and Travora exists as well.

tarantonio commented:
@ninjalulz seems that you are working on Oramond map, maybe it's better to wait until you add it to update minimap.

pitis91 commented:
roshamuul also have now full north-west missing valley but also is missing the mines from north-east and missing parts of roshamuul quest under floors.

Royalot committed:
update minimap

changed the already included areas
![minimap_floor](https://cloud.githubusercontent.com/assets/6708725/6971325/151e0e8c-d97c-11e4-8887-4f8015cc6cee.png)
